src/lib/api/patrons/patron.js

```diff
@@ -14,17 +16,63 @@ const get = async patronPid => {
   return response;
 };
 
-const list = async queryText => {
-  const response = await http.get(`${listUrl}?q=${queryText}*`);
+const list = async (queryText, wildcard = true) => {
+  const response = await http.get(
+    `${listUrl}?q=${queryText}${wildcard ? '*' : ''}`
```

Comment:
Did you understand why this `*` is needed in the end? Is it for the `keywords` fields?

Reply:
A new issue in place, so we can close this pr and not mix things, https://github.com/inveniosoftware/invenio-app-ils/issues/922